lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -136,6 +130,9 @@ export abstract class ClientApplication {
         return silentFlowClient.acquireToken(this.initializeRequestScopes(request) as SilentFlowRequest);
     }
 
+    /**
+     * Gets the cache manager for the application.
```

Comment:
This should actually be getTokenCache(), instead of getCacheManager(). I will submit a different PR updating. For the comment, can we do "Gets the token cache for the application"

Reply:
Sure, comment updated.